tests/testthat/test-sort_hierarchical.R

```diff
@@ -68,12 +68,46 @@ test_that("sort_hierarchical(sort = 'alphanumeric') works", {
   )
 })
 
+test_that("sort_hierarchical(sort) works with different sorting methods for each variable", {
```

Comment:
This is wonderful. Can you please add tests that include calls to `add_overall()` as well?